src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php

```diff
@@ -80,6 +80,12 @@ public function getOpenFileCases()
             array('../TwigBundle/README.md', false),
             array('Controller/../README.md', false),
             array('Controller/./ProfilerController.php', false),
+            array('app/config/parameters.yml', false),
+            array('config/services.yaml', false),
+            array('var/anything', false),
+            array('VAR/anything', false),
+            array('app/Resources/views/base.html.twig', true),
+            array('app/Resources/SomeBundle/views/some.html.twig', true),
```

Comment:
If you exclude the old templates location, maybe `/templates` should be excluded too?

Reply:
Ah, my bad, you're not excluding the templates. I read that wrong.